### PR TITLE
Metabase tutorial

### PR DESCRIPTION
**Pull request by matkuliak**
## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/integrations/metabase.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- Database type (PostgreSQL)
+- Display name
+- Host (the URI of your cluster)
+- Database name
+- Username
+- Password
+
+.. NOTE::
+
+    If you choose to use Metabase with a Cloud cluster, also select the "Use
```
Comment (SStorm) [inline docs/integrations/metabase.rst]: SSL is pretty normal in all clusters, perhaps we should rephrase this to "Make sure you select "Use a secure connection (SSL)" option, unless your cluster is not configured for SSL".

**Review by SStorm — APPROVED** [review]

File: docs/integrations/metabase.rst (hunk 1 above)
Comment (matkuliak) [reply docs/integrations/metabase.rst]: Makes sense, thank you!
